### ISSUE TITLE
TTable incomplete type errors in ROOT6 tests

### ISSUE THREAD

**Issue by plexoos** [star-bnl/star-sw#594]
There are errors in ROOT6 tests like this one:
```
In file included from Table_Cint dictionary payload:21:
LoadTable: .L /star-sw/StarDb/Geometry/tpc/tpcPadConfig.y2019.C
In file included from /star-sw/.sl79_gcc485/include/TGenericTable.h:8:
/star-sw/.sl79_gcc485/include/TTableDescriptor.h:25:33: error: base class has incomplete type
class TTableDescriptor : public TTable {
                         ~~~~~~~^~~~~~
/star-sw/.sl79_gcc485/include/TTable.h:48:7: note: definition of 'TTable' is not complete until the closing '}'
class TTable : public TDataSet {
      ^
```

It is likely because I removed the TTable from ROOT6.16 and the external `star-table` (https://github.com/star-bnl/star-table) from the environment hoping that StRoot/Table would provide the needed functionality. Clearly, it does not. Who is maintaining StRoot/Table?

_Originally posted by @plexoos in https://github.com/star-bnl/star-sw/issues/592#issuecomment-1741484400_

**Comment by veprbl** [comment]
The difference might be coming from the way the dictionaries are generated. Why not keep the star-table in place?

**Comment by plexoos** [comment]
That's exactly how I hope to solve this

**Comment by plexoos** [comment]
resolved by #595
